kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala

```diff
@@ -39,18 +40,17 @@ private[v1] class OperationsResource extends ApiRequestContext {
     content = Array(new Content(
       mediaType = MediaType.APPLICATION_JSON)),
     description =
-      "Get an operation detail with a given session identifier and operation identifier")
+      "Get an operation event")
   @GET
   @Path("{operationHandle}")
-  def getOperationDetail(
-      @PathParam("operationHandle") operationHandleStr: String): OperationDetail = {
+  def getOperationEvent(
+      @PathParam("operationHandle") operationHandleStr: String): KyuubiEvent = {
     try {
-      val operation = backendService.sessionManager.operationManager
-        .getOperation(parseOperationHandle(operationHandleStr))
```

Comment:
```scala
val operation = backendService.sessionManager.operationManager
        .getOperation(parseOperationHandle(operationHandleStr))
KyuubiStatementEvent(operation)
````

And we can refine KyuubiStatementEvent(statement: KyuubiOperation)

Reply:
The type of `operation` in UT is NoopOperation,  cannot construct KyuubiStatementEvent directly.

Reply:
We shall build some tests on the real operation now